packages/headless/src/integration-tests/commerce-placements.test.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @cspell/spellchecker */
+import {buildCommercePlacementsEngine} from '../app/commerce-placements-engine/commerce-placements-engine';
+import {
+  getBadge,
+  getRecs,
+} from '../features/placement-set/placement-set-action';
+import {getOrganizationEndpoints} from '../recommendation.index';
+
+describe('commerce placements', () => {
```

Comment:
Some simple end to end tests just to verify that the boilerplate is working.

As explained in the channel however, I am simply unable to perform a request with the "correct values" that would actually return recommendations or badge: the request is not error-ing, but there's no products or badges that ever get returned.

I assume it's due to incorrect parameters, but so far, I haven't been able to "guess" the correct combination that would return something meaningful.

Reply:
This is probably because there isn't any badging or recs configured on the property. Maybe we can use the barca org in dev? It should have some recs and badging configured in the merch hub.

Otherwise, we might want to create a test org for the unified api. This org could be used in unified commerce api functional tests as well.